js/src/dapps/dappreg/_colors.css

```diff
@@ -15,5 +15,4 @@
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-.app {
-}
+$blue: rgb(41, 128, 185);
```

Comment:
Would have made the name "title-background" or "title-color" (something specific, not tied to the color but actual usage thereof)